libs/community/langchain_community/document_loaders/parsers/pdf.py

```diff
@@ -46,6 +58,119 @@
     "JBIG2Decode",
 ]
 
+logger = logging.getLogger(__name__)
+
+_format_image_str = "\n\n{image_text}\n\n"
+_join_images = "\n"
+_join_tables = "\n"
+_default_page_delimitor = "\n\f"
+
+
+def purge_metadata(metadata: dict[str, Any]) -> dict[str, Any]:
```

Comment:
could we keep it private if it's only used in this file?

```suggestion
def _purge_metadata(metadata: dict[str, Any]) -> dict[str, Any]:
```